ipykernel/debugger.py

```diff
@@ -600,11 +600,14 @@ async def modules(self, message):
         mods = []
         for i in range(startModule, moduleCount):
             module = modules[i]
-            filename = getattr(getattr(module, "__spec__", None), "origin", None)
-            if filename and filename.endswith(".py"):
-                mods.append({module.__name__: filename})
-
-        reply = {"body": {"modules": mods, "totalModules": len(modules)}}
+            filename = getattr(getattr(module, '__spec__', None), 'origin', None)
+            if filename and filename.endswith('.py'):
+                mods.append({
+                    'id': module.__name__,
+                    'name': filename
```

Comment:
From the screenshot [there](https://github.com/jupyterlab/jupyterlab/issues/11966#issuecomment-1026031708), it seems more appropriate to use:

```python
{
  'id': <incremental integer>,
  'name': module.__name__,
  'path': filename
}
```

Reply:
Done. Thx @fcollonval